packit/config/package_config.py

```diff
@@ -150,6 +151,11 @@ def get_from_dict(
             else:
                 raise PackitConfigException("Spec file was not found!")
 
+        if not os.path.isfile(raw_dict["specfile_path"]):
```

Comment:
or

```suggestion
        if not Path(raw_dict["specfile_path"]).is_file():
```

so you don't have to import the `os`